astro/src/content/docs/get-started/start-here/step-3.mdx

```diff
@@ -0,0 +1,84 @@
+
+This process is also know as access control. It is the process of controlling the users that have access to specific pages and functions of an application. The first step is to protect a page such that only users that have logged in can view it.
+
+Open the file `src/index.mts` and find the route named `/account`. It should look something like this:
+
+```typescript
+app.get("/account", async (req, res) => {
+  if (!await sdk.userHasAccess(req, res, ["admin", "user"])) {
+    res.redirect(302, '/login');
+    return;
+  }
+
+  res.sendFile(path.join(__dirname, '../templates/account.html'));
+});
+```
+
+The first check is where the application determines if the user is logged in or not. This is done by calling the SDK function `userHasAccess` (you can ignore the last parameter, which we will cover in the next section). This function returns true if the user has access, which implicitly verifies that they are logged in. Let's take a look at this function.
+
+Open the file `src/sdk.ts` and find the function named `userHasAccess`. One note is that this application uses `jose` which is a Node library used for validating and parsing [JSON Web Tokens (JWTs)](/docs/lifecycle/authenticate-users/login-api/json-web-tokens). If you want to learn more about `jose`, you can visit the project at Github here: https://github.com/panva/jose
+
+The `userHasAccess` method begins by loading the user from the request. This is accomplished by calling the `getUser` function of the SDK, which returns the user's access token. If the access token comes back `null`, that means the user is not logged in and this function returns false.
+
+```typescript
+async userHasAccess(req: Request, res: Response, roles: Array<string>): Promise<boolean> {
+  const jwt = await this.getUser(req, res);
+
+  // ...
+}
+```
+
+Let's take a quick peek at the `getUser` function of the SDK. This function looks like this:
+
+```typescript
+// 1. Load the user's access token from a cookie in the request
+let accessToken = req.cookies[this.configuration.accessTokenCookieName];
+if (!accessToken) {
+  return null;
+}
+
+// 2. Verify that the access token is valid
+let payload: JWTPayload;
+try {
+  payload = (await jose.jwtVerify(accessToken, this.JWKS, {
+    issuer: this.configuration.oauthIssuer,
+    audience: this.configuration.applicationId,
+  })).payload;
+} catch (e) {
+  // ...
+}
+```
+
+The first part of this function loads a cookie from the request that contains the user's access token. The access token is part of the OAuth specification, and is generated after the user successfully logs in. FusionAuth uses JWTs for access tokens. 
+
+The second part of this function verifies that the JWT is valid using `jose`'s `jwtVerify` function. This function validates the cryptographic signature of the JWT and also ensures that the JWT is not expired and all of the claims are valid.
```

Comment:
```suggestion
The second part of this function verifies that the JWT is valid using `jose`'s `jwtVerify` function. This function validates the cryptographic signature of the JWT and also ensures that the JWT is not expired and the issuer and audience claims are expected values.
```

Reply:
I'm thinking about rejecting this change. We haven't described these claims and I think it will confuse the reader unless they already know about JWTs or have read our JWT doc.